chapters/synchronization.txt

```diff
@@ -53,67 +53,133 @@ _memory dependencies_ between two sets of operations defined by the
 command's two _synchronization scopes_.
 
 [[synchronization-dependencies-scopes]]
-The synchronization scopes define which other operations a synchronization
-command is able to create execution dependencies with.
-Any type of operation that is not in a synchronization command's
+A synchronization scope is a set of selection criteria for operations.
```

Comment:
Not super keen on this wording but I see where you're going. Maybe this:

```suggestion
A synchronization scope identifies a set of operations relative to a synchronization command.
```

?

Reply:
To my mind, these two options are not just different wordings, but actually express two different concepts.

I.e. is a synchronization scope the criteria or is it the operations?

In my opinion, either could work (with some refinements to the wording) but they have some different pros/cons.

Reply:
@Tobski Yea, I try superhard to avoid implying here that scopes are operation. If scopes did "identify set of operations" that basically implies it is an identity with set of operations. I don't want it to be set of operations at all. I want it sort of be a SQL query instead. Which can later on access a set of operations and filter them down.

Reply:
I was trying to state these as the "identifier" rather than the "set of operations" but I see the confusion. I haven't got a better idea so let's just leave it for now.